src/components/search/content-highlights/ContentHighlightSet.jsx

```diff
@@ -22,13 +22,36 @@ const ContentHighlightSet = ({ highlightSet }) => {
     title,
   } = highlightSet;
 
-  const transformedTighlightedContent = useMemo(() => highlightedContent.map(highlightedContentItem => (
+  const archivedContent = useMemo(() => [], []);
+  const activeContent = useMemo(() => [], []);
+  for (let i = 0; i < highlightedContent.length; i++) {
+    const {
+      courseRunStatuses,
+    } = highlightedContent[i];
+    if (courseRunStatuses) {
+      if (courseRunStatuses?.every(status => status === 'archived')) {
```

Comment:
nit: Can probably be a helper method since we're checking for this a lot

Reply:
Seems like more code than just a one line method though

Reply:
True, but if the logic gets more complicated in the future, having it in a helper method makes it easier to track down and update.

At the very least, can make `archived` a constant.